pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -78,6 +78,9 @@ func NewCmdStartBuild(fullName string, f *clientcmd.Factory, out io.Writer) *cob
 
 	cmd.Flags().String("git-post-receive", "", "The contents of the post-receive hook to trigger a build")
 	cmd.Flags().String("git-repository", "", "The path to the git repository for post-receive; defaults to the current directory")
+
+	cmdutil.AddOutputFlagsForMutation(cmd)
```

Comment:
Omit start build from this pr because this breaks a lot of examples and tests that expect the output to be the name only.  We'll need a more nuanced solution there.